packages/powershell/oh-my-posh/oh-my-posh.psm1

```diff
@@ -15,7 +15,10 @@ function Get-PoshCommand {
         }
         return "$PSScriptRoot/bin/posh-linux-amd64"
     }
-    if ([Environment]::Is64BitOperatingSystem) {
+    if ((Get-CimInstance Win32_OperatingSystem).OSArchitecture -icontains "arm") {
+        return "$PSScriptRoot/bin/posh-windows-arm.exe"
```

Comment:
this binary isn't part of the module. In fact, it's not even working atm.

Reply:
OK. I just want to help. Follow the Tutorial: Set up Powerline in Windows Terminal would receive errors in Windows Arm.
I tried x86 build. It is very slow in git directory (and git is also 32-bit). Then I build arm binary for better performance. But its just arm 32-bit. Golang only support armv7 in Windows.

Reply:
I've been trying to get it to cross compile but Windows isn't supporting ARM properly (you can't even install it on the latest raspberry PI's to setup a build system). I want to support it, but Windows is giving me a hard time. So I'm afraid I can't accept this until we can also create this binary during our CD pipeline.